sympy/discrete/transforms.py

```diff
@@ -116,7 +169,8 @@ def fft(seq, dps=None):
 
     """
 
-    return _fourier_transform(seq, dps=dps)
+    return _fourier_transform(seq, dps=dps,
+    fourier_parameters=fourier_parameters)
```

Comment:
A few tabs (4 spaces) would be preferable below the return statement before the keyword argument.